plugin.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0"
+        id="edu.berkeley.eecs.emission.cordova.integritydetect"
+        version="1.0.0">
+
+  <name>IntegrityDetection</name>
+  <description>Integrity detection for road surface </description>
+  <author>Bingbing Huang (bing2013@cs.washington.edu)</author>
+  <license>Modified BSD license</license>
+
+  <engines>
+    <engine name="cordova" version=">=3.6.0"/>
+  </engines>
+
+  <platform name="android">
+    <config-file target="res/xml/config.xml" parent="/*">
+      <feature name="integrityDetect">
+        <param name="android-package" value="edu.berkeley.eecs.emission.cordova.integrityDetect.IntegrityNotificationReceiver"/>
+      </feature>
+    </config-file>
+
+      <config-file target="AndroidManifest.xml" parent="/manifest/application">
+        <receiver
+            android:name="edu.berkeley.eecs.emission.cordova.integrityDetect.IntegrityNotificationReceiver"
+            android:enabled="true">
+            <intent-filter>
+                <action android:name="local.transition.initialize"></action>
+                <action android:name="local.transition.exited_geofence"></action>
+                <action android:name="local.transition.stopped_moving"></action>
+            </intent-filter>
+        </receiver>
+    </config-file>
+
```

Comment:
you also need to add the service to the manifest, otherwise it won't start. Again, use 
`plugins/edu.berkeley.eecs.emission.cordova.datacollection/plugin.xml` as your example and follow the receiver/service pattern exactly